File: ascii-doc-validator/tests/code/java/UserController.java
package com.example.controllers;

import com.example.models.User;
import java.util.*;

public class UserController {
    private Map<Long, User> db = new HashMap<>();

    public User getUser(Long id) {
        return db.get(id);
    }

    public void createUser(Long id, String name) {
        db.put(id, new User(id, name));
    }
}

File: ascii-doc-validator/tests/code/python/user_controller.py
from typing import Dict, Optional
from models import User

class UserController:
    """Контроллер для управления объектами User."""

    def __init__(self):
        self.db: Dict[int, User] = {}

    def get_user(self, user_id: int) -> Optional[User]:
        return self.db.get(user_id)

    def create_user(self, user_id: int, name: str) -> None:
        self.db[user_id] = User(user_id, name)